portfolio/src/main/webapp/script.js

```diff
@@ -28,13 +30,22 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
 /**
  * Displays a string fetched from a servlet.
  */
 async function displayMessage() {
     const responseFromServer = await fetch('/fetch-hello-world-string');
-    const textFromResponse = await responseFromServer.text();
+    
+    // Saves JSON list
+    const jsonList = await responseFromServer.json();
+    console.log(jsonList);
+    
+    // Selects a random string from the JSON list
+    const randomMessage = jsonList[Math.floor(Math.random() * jsonList.length)];
+    console.log(randomMessage);
```

Comment:
Don't include log messages in PRs unless its a placeholder and meant to be removed in a future PR.

Reply:
Got it, I'll make the changes.